src/components/TVChartContainer/charting_library/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
do we need this?

Reply:
I will remove them. I left them to some predefined types. We are not using them right now. If needed we will create.

Reply:
I will remove